automol/util/vector.py

```diff
@@ -154,112 +152,101 @@ def unit_perpendicular(
     :type value_if_parallel: Vector
```

Comment:
The inputs for `unit_perpendicular` need types (`Sequence | Vector` for all of them).

Reply:
Also, we'll need to `require_vector_like` for the inputs.